attributes/default.rb

```diff
@@ -130,8 +130,27 @@
   default['kibana']['kibana_service'] = "http://#{node['kibana']['interface']}:#{node['kibana']['port']}"
 end
 
-#<> The kibana service configuration source
-default['kibana']['service']['source'] = 'upstart.conf.erb'
+# kibana service configurations - defaults to settings for Ubuntu 14.04
+case node['platform']
+when 'centos'
+  if node['platform_version'] < '6.9'
+    default['kibana']['service']['source'] = 'initd.kibana.erb'
+    default['kibana']['service']['template_file'] = '/etc/init.d/kibana'
+  else
+    default['kibana']['service']['provider'] = Chef::Provider::Service::Systemd
```

Comment:
The Travis build fails now - it appears the Ubuntu 16.04 travis test tries to use Init instead of Systemd as the default provider...